src/System.IO.FileSystem/src/System/IO/File.cs

```diff
@@ -386,6 +403,76 @@ private static String InternalReadAllText(String path, Encoding encoding)
                 return sr.ReadToEnd();
         }
 
+
+        // If we use the path-taking constructors we will not have FileOptions.Asynchronous set and
+        // we will have asynchronous file access faked by the thread pool. We want the real thing.
+        private static StreamReader AsyncStreamReader(string path, Encoding encoding)
+        {
+            FileStream stream = new FileStream(
```

Comment:
@JonHanna, it rather surprised me when I found that it only searches in the `default` (usually `master`) branch:

> Only the default branch is considered. In most cases, this will be the master branch.

from https://help.github.com/articles/searching-code/

Sometimes, the indexing is bit lagging, but that code is present in current master. :-)

note: when creating the link with line numbers, I froze the commit SHA-1 to tip of the current master to keep it future proof.